docker/Dockerfile.base

```diff
@@ -18,16 +20,13 @@ ENV DEBIAN_FRONTEND=noninteractive
 # Install packages for apt repo
 RUN apt-get -qq update \
     && apt-get upgrade -y \
-    && apt-get -qq install --no-install-recommends -y \
-    gnupg wget unzip tzdata nginx libnginx-mod-rtmp \
```

Comment:
removing `nginx` and `libnginx-mod-rtmp` as they are built-in to our compiled version of NGINX along with `nginx-vod-module`.